chef/cookbooks/glance/recipes/cache.rb

```diff
@@ -37,6 +37,14 @@
   cinder_api_insecure = cinder[:cinder][:api][:protocol] == 'https' && cinder[:cinder][:ssl][:insecure]
 end
 
+glance_stores = ["glance.store.filesystem.Store",
+                 "glance.store.http.Store",
+                 "glance.store.cinder.Store",
+                 "glance.store.rbd.Store",
+                 "glance.store.swift.Store"]
+
+glance_stores << "glance.store.vmware_datastore.Store" unless node[:glance][:vsphere][:host].empty?
+
```

Comment:
I've moved `glance_store` to default attributes. @dirkmueller : Is it ok for you?